src/main/java/mate/academy/security/jwt/JwtTokenProvider.java

```diff
@@ -20,7 +20,7 @@
 @Component
 public class JwtTokenProvider {
     @Value("${security.jwt.token.secret-key:secret}")
-    private String secretKey;
+    protected String secretKey;
```

Comment:
Consider changing the access modifier of 'secretKey' back to 'private' to maintain encapsulation and prevent unintended access or modifications from other classes.